src/views/Setup.vue

```diff
@@ -90,7 +93,15 @@ onUnmounted(() => clearInterval(waitingForRegistrationInterval));
         <!-- Step two - setup space controller -->
         <SetupController
           v-else-if="route.params.step === '2' && ensAddress"
```

Comment:
Can't we have more expressive route params? `/setup/controller` instead of `/setup/2`